src/pages/RulesPage.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import RulesComponent from "../components/RulesPage";
+
+export const RulesPage = () => (
+    <div>
```

Comment:
```suggestion
    <div className={classes.rulesCard}>
```

Reply:
Is this solved with the rebase?

Reply:
Maybe (?)

I looked at the preview, and the only thing I think is missing is a padding with `spacing(4)`. So maybe add that class with it anyway.